develop/learning-paths/migration/articles/01-intro/01-from-a-monolith-to-a-modular-architecture.markdown

```diff
@@ -0,0 +1,124 @@
+# From a Monolith to a Modular Architecture
+
+Since Liferay Portal's beginning it's been an amazing application full of all
+kinds of capabilities. It's had powerful built-in apps and extensible frameworks
+that deploy on all kinds of devices. These apps leverage Portal's powerful
+frameworks (e.g., asset and comment frameworks) and in some cases the apps
+leverage each other (e.g., the Blogs and Wiki apps leverage the Documents and
+Media Library). And undoubtedly you've leveraged Liferay Portal's apps and
+frameworks via their APIs. Yes, Liferay Portal has traditionally provided so
+many things to so many users and developers--and that's been a very good thing.
+
```

Comment:
You'll want to vary your adjectives and verbs here. You've got several uses of "powerful" and "leverage," as well as other superlatives (amazing, all kinds of, etc.). The audience here is the Liferay developer upgrading his knowledge; we don't need to sell him on how awesome Liferay is.